src/service/worker/runtime/typings.ts

```diff
@@ -184,6 +191,7 @@ export interface RawServiceJSON {
   timeout?: number,
   runtimeArgs?: string[],
   routes?: Record<string, ServiceRoute>,
+  globalRateLimit?: Record<string, RateLimiter>
```

Comment:
I think we should have well-defined supported keys here, only `events` and `routes`.

Maybe this would work? 
```suggestion
  globalRateLimit?: Pick<Record<string, RateLimiter>, 'events' | 'routes'>
```

If not, we may need to declare a custom `GlobalRateLimiter` interface which has only the actual fields we support.

Reply:
Also just not to forget, reminder that the global rate limiter still needs to be plugged in the pipeline